distribute/files/VisiCut.MacOS

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
-DIR="$( cd -P "$( dirname "$0" )" && pwd )"
-
-# Print and check java version
-java -version 2>&1 | tee /dev/stderr | egrep -q '"[1-9][0-9]\..*"' || echo "Warning: Your Java version may be too old. We recommend Java 11 or newer. Trying anyway."
-
-java -Xms256m -Xmx2048m -jar "$DIR/Visicut.jar" "$@"
+bundle_identifier="$(osascript -e 'id of app "VisiCut"')"
+if [ $? -eq 0 ] 
```

Comment:
If both the app and the ZIP file version are present, then the ZIP version would always start the app. I don't think that is intended.

Reply:
I think its the opposite way. The MacApp Version is preferred over the ZIP version if an MacApp is found. I will try to change this in a way that it will always use the version from which the extension is installed.